.github/workflows/sample.yml

```diff
@@ -0,0 +1,27 @@
+on: [push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 8.x
+      - name: Install root project dependencies
+        run: npm install
+      - name: Install sample project dependencies
+        run: npm install
+        working-directory: ./sample
+      - name: Compile sample project
+        run: npm run compile
+        working-directory: ./sample
+      - name: Run tests
+        uses: GabrielBB/xvfb-action@v1.0
```

Comment:
This is forcing use of v1 which is more than two years old. Is that a mistake or on purpose, especially as current version is `v1.6. Using a movable tag like `v1` would avoid this IMHO.